charts/netbox/templates/configmap.yaml

```
@@ -126,6 +127,7 @@ data:
     ENFORCE_GLOBAL_UNIQUE: {{ toJson .Values.enforceGlobalUnique }}
     EXEMPT_VIEW_PERMISSIONS: {{ toJson .Values.exemptViewPermissions }}
     FIELD_CHOICES: {{ toJson .Values.fieldChoices }}
+    FILE_UPLOAD_MAX_MEMORY_SIZE: {{ int fileUploadMaxMemorySize }}
```

Looks like this was missing a `.Values.`:

```suggestion
    FILE_UPLOAD_MAX_MEMORY_SIZE: {{ int .Values.fileUploadMaxMemorySize }}
```